merlin-server/constraints-dsl-compiler/src/libs/constraints-edsl-fluent-api.ts

```diff
@@ -697,6 +705,10 @@ declare global {
     /**
      * Invert all the windows produced this.
```

Comment:
Same as above: I think this is supposed to be attached to `public not() Windows;` below.

Reply:
same as above, this is fixed in commit 3